tests/Oryx.BuildImage.Tests/DotNetCore/DotNetCoreDynamicInstallTest.cs

```diff
@@ -737,7 +737,7 @@ public void DotnetFails_ToInstallUnsupportedSdk(string runtimeVersion, string im
                 () =>
                 {
                     Assert.False(result.IsSuccess);
-                    Assert.Contains($"Error: Platform '{DotNetCoreConstants.PlatformName}' version '{runtimeVersion}' is unsupported.", result.StdErr);
+                    Assert.Contains($"Debug mode enabled\nPlatform '{DotNetCoreConstants.PlatformName}' version '{runtimeVersion}' is unsupported.", result.StdErr);
```

Comment:
We should figure out why `Debug mode enabled` is being displayed without us providing the `--debug` flag in the command above (as far as I can tell we aren't providing the flag, but it may be provided under-the-hood from one of our helper methods called on `ShellScriptBuilder`)

This comment applies to all of the other tests that have this same issue.

Reply:
yea it seems like --debug was provided in `[ShellScriptBuilder](https://github.com/microsoft/Oryx/blob/main/src/BuildScriptGenerator.Common/ShellScriptBuilder.cs#L62-L65)`. But it's interesting why it didn't show up in the past.

Reply:
Found the reason why debug message went into error stream. I changed console.WriteLine to console.Error.WriteLine for outputing debug enable message. Shoudl be fixed now.